sdk/cosmos/tests.yml

```diff
@@ -36,29 +36,3 @@ stages:
       TestGoals: 'verify'
       TestOptions: '$(ProfileFlag) $(AdditionalArgs)'
       TestResultsFiles: '**/junitreports/TEST-*.xml'
-
```

Comment:
Why are we removing this? Can't we still run spring-data-cosmos-test integration pipelines on `azure-cosmos` module development?

Reply:
+1 - same comment for the ci.yml above. We would still need to compile and run the spring-data-cosmos tests for any changes in azure-cosmos